examples/app.py

```diff
@@ -35,35 +51,121 @@
 
 You can find the database in `examples/app.db`.
 
-Run the development server:
+
+3. Look for a MARC21 file. In our case, we are going to use `authority.xml`
+   which is included in the package `invenio_records`.
+
+.. code-block:: console
+
+    $ demomarc21pathname=$(echo "from __future__ import print_function; \
+      import pkg_resources; \
+      print(pkg_resources.resource_filename('invenio_records', \
+      'data/marc21/authority.xml'))" | python)
+
+
+4. Transform the file `authority.xml` (Path stored in `demomarc21pathname`
+   variable) from MARC21 to a json file:
+
+.. code-block:: console
+
+    $ dojson do -i $demomarc21pathname -l marcxml marc21 > records.json
+
+
+5. Load the file in the application importing the json file:
 
 .. code-block:: console
 
-    $ flask -a app.py run -p 5000 -h '0.0.0.0'
+    $ flask -a app.py records create records.json
+
+
+6. Create a root collection using the CLI. This collection will become
+   the root of new tree by default:
+
+.. code-block:: console
+
+    $ flask -a app.py collections create Demo
```

Comment:
If you create collections before uploading records you can search in `_collections` field in record metadata.

--
tip: make sure that `COLLECTIONS_REGISTER_RECORD_SIGNALS=True`